products/sle/main.pm

```diff
@@ -949,20 +949,24 @@ elsif (have_scc_repos()) {
     }
 }
 elsif (get_var('HPC')) {
-    if (check_var('HPC', 'install')) {
-        load_boot_tests();
-        load_inst_tests();
-        load_reboot_tests();
+    if (get_var('CLUSTER_NODES')) {
+        load_cluster_boot();
+    }
+    else {
+        loadtest 'boot/boot_to_desktop';
+        loadtest "console/hostname";
+    }
+    loadtest 'hpc/before_test' if (sle_version_at_least('15'));
```

Comment:
Maybe you can use the new version `is_sle('15+')` instead of `sle_version_at_least`, no?

Reply:
I think this change should be done to don't impact QAM flow:
loadtest 'hpc/before_test' if (sle_version_at_least('15') && !is_updates_tests);